### PR TITLE
feat: add initial package contents

### PR DESCRIPTION
**Pull request by abutterworth**
The initial set of files to include in this default branding package.

Refer to the README in this PR for detail.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,6 +1,43 @@
 Open edX Brand Package Interface
 ================================
 
-This project contains the default branding and style used in Open edX applications.
+This project contains the default branding assets and style used in Open edX applications. It is published on npm as `@edx/brand-openedx`.
 
 The file structure serves as an interface to be implemented for custom branding and theming of Open edX.
+
+This project contains branding assets and themes for edx.org.
```
Comment (stvstnfrd) [inline README.rst]: Open edX?

Comment (abutterworth) [reply README.rst]: Thanks I removed the whole sentence since it was duplicative.

File: README.rst [hunk 2]
```diff
@@ -1,6 +1,41 @@
...
-This project contains the default branding and style used in Open edX applications.
+This project contains the default branding assets and style used in Open edX applications. It is published on npm as `@edx/brand-openedx`.
 
 The file structure serves as an interface to be implemented for custom branding and theming of Open edX.
+
+-----------------------
+How to use this package
+-----------------------
+
+Applications in Open edX a configured by default to include this package for branding assets and theming visual style.
```
Comment (AlasdairSwan) [inline README.rst]: Should be ```Applications in Open edX are configured```

File: README.rst [hunk 3]
```diff
@@ -1,6 +1,41 @@
...
+
+To use a custom brand and theme...
+
+1. Fork or copy this project. Ensure that it lives in a location accessible to Open edX applications during asset builds. This may be a publish git repo, npm, or local folder depending on your situation.
```
Comment (AlasdairSwan) [inline README.rst]: Should be ```This may be a published git repo```

File: README.rst [hunk 4]
```diff
@@ -1,6 +1,41 @@
...
+
+.. image:: /logo.svg
+    :alt: logo
+    :width: 128px
+
+``/logo-trademark.svg`` A variant of the logo with a trademark ® or ™. Note: This file must be present. If you don't have a trademark variant of your logo, copy your regular logo and use that.
+
+.. image:: /logo-trademark.svg
+    :alt: edX
+    :width: 128px
```
Comment (AlasdairSwan) [inline README.rst:38]: These images aren't loading when I view the rendered version.

File: README.rst [hunk 5]
```diff
@@ -1,6 +1,41 @@
...
+
+To use a custom brand and theme...
+
+1. Fork or copy this project. Ensure that it lives in a location accessible to Open edX applications during asset builds. This may be a publish git repo, npm, or local folder depending on your situation.
+
+2. Replace the assets in this project with your own logos or SASS theme. Match the filenames exactly. Open edX applications refer to these files by their filepath. Refer to the brand for edx.org at https://github.com/edx/brand for an example.
+
+3. Configure your Open edX instance to consume your custom brand package. Refer to this documentation on configuring the platform [TODO: Add a link to relevant documentation].
+
+4. Rebuild the assets and microfrontends in your Open edX instance to see the new brand reflected. [TODO: Add link to relevant documentation].
```
Comment (AlasdairSwan) [inline README.rst]: I assume you know these links are still outstanding.

**Review by AlasdairSwan — CHANGES_REQUESTED** [review]
Some type-os and broken images.

File: README.rst (hunk 4 above)
Comment (abutterworth) [reply README.rst:38]: I think these work when it's merged. The path location is the same as it is in @edx/brand-edx.org

**Review by AlasdairSwan — APPROVED** [review]
:shipit:

**Pull request merged**